### PR TITLE
Created a form for adding new project, CSSed it

### PR DESCRIPTION
**Pull request by Liliya-V**

### CODE REVIEW

File: adminPanel/addForm.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=PortfolioProjects", 'root', '');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
```
Comment (alecorra) [inline adminPanel/addForm.php]: delete empty lines

File: adminPanel/addForm.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+?>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina</title>
```
Comment (alecorra) [inline adminPanel/addForm.php]: change page's title, it's not the portfolio

**Review by alecorra — COMMENTED** [review]
only a couple of suggestion, the rest is good

File: adminPanel/addForm.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="../css/styleAdminPanel.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <h1>Welcome to a project adding page, Liliya</h1>
+        <form  action="addformDB.php" method="POST">
+            <div class="container">
+            <label for="title">Type a title here:</label>
```
Comment (mporam) [inline adminPanel/addForm.php]: fix indentation here, container is a parent so everything inside it should be indented

File: adminPanel/addformDB.php [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (mporam) [inline adminPanel/addformDB.php]: delete this file as it is doing nothing at the moment

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**